flutter/Graminsta/lib/auth/login.dart

```diff
@@ -1,11 +1,46 @@
 import 'package:flutter/material.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:Graminsta/http.dart';
+import 'package:dio/dio.dart';
 
 class LoginPage extends StatefulWidget {
   @override
   _LoginPageState createState() => new _LoginPageState();
 }
 
 class _LoginPageState extends State<LoginPage> {
+  final _usernameController = TextEditingController();
+  final _passwordController = TextEditingController();
+
+  void login() async {
+    Response response;
+    HttpManager.post("core/login/", data: {
+      "username": _usernameController.text,
+      "password": _passwordController.text,
```

Comment:
The password should be hashed for security consideration.
https://stackoverflow.com/questions/47870642/dart-md5-from-string

Reply:
Actually, I think you mean encryption. Yeah, that's a technique to prevent password intercepted, but by default, Django expects real password instead of encrypted one, the hashing of the password is computed during the Django authentication process, for different password, the `salt` and the `hashing algorithm` might be different. And back to this, I think the best thing you can do is let client (Flutter app in this case) to encrypt the password with some public key, and server decrypts the encrypted password with its private key, but it's kind of overkill for a starter project. Usually, if you are using HTTPS, it should provide enough security for this case, as the encryption and decryption is done by SSL. So I'd suggest we just leave this as it is now.